<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>380</y>
      <width>502</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="btnsLayout">
     <item>
      <widget class="QPushButton" name="setBtn">
       <property name="text">
        <string>Set array</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mergeBtn">
       <property name="text">
        <string>Merge</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="heapBtn">
       <property name="text">
        <string>Heap</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quickBtn">
       <property name="text">
        <string>Quick</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="interpolBtn">
       <property name="text">
        <string>Interpolation</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
